app/src/components/dashboard.scss

```diff
@@ -0,0 +1,34 @@
+// should I be adding new vriables here or upstream in the index.scss so that other
+// components can also access them :?
+$scarlet: #f05020;
```

Comment:
I think it could make sense to create a global scss file called either base.scss or colors.scss.

Reply:
cool. I moved it all to index.scss